CoreOpenGLDisplay/src/au/gov/asd/tac/constellation/visual/opengl/utilities/glyphs/DefaultLabelFonts.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2010-2019 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.visual.opengl.utilities.glyphs;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.openide.util.lookup.ServiceProvider;
+
+/**
+ * The default fonts for use as labels in a graph.
+ *
+ * @author cygnus_x-1
+ */
+@ServiceProvider(service = ConstellationLabelFonts.class, position = Integer.MAX_VALUE)
+public class DefaultLabelFonts implements ConstellationLabelFonts {
+
+    @Override
+    public List<String> getFontList() {
+        final List<String> fontList = new ArrayList<>();
+//        fontList.add("SansSerif");
```

Comment:
Even though the font will fall back to Sans Serif, I think there's value in making it explicit here, in particular so that it shows up in the UI.